modules/mono/csharp_script.cpp

```diff
@@ -1579,37 +1577,33 @@ bool CSharpScript::_update_exports() {
 	return false;
 }
 
-bool CSharpScript::_update_signals() {
-	if (!valid)
-		return false;
-
-	bool changed = false;
-
-	if (signals_invalidated) {
-		signals_invalidated = false;
+void CSharpScript::load_script_signals(GDMonoClass *scrp_class) {
 
-		GDMonoClass *top = script_class;
+	// no need to load the script's signals more than once
```

Comment:
Correct me if I'm wrong but this will prevent signal list to be updated in the editor when building after some code changes, as `signals_invalidated` is never reverted to `true`.  

Otherwise thanks for getting on this issue when I can't :+1: :+1: :+1:

Reply:
Thanks for the suggestion, I re-added that check in the `reload_assemblies_if_needed` method before `reload` gets called now.